name-context.txt

```diff
@@ -604,3 +604,8 @@ w/r/t the collection of snapdragons growing wild in cracks surrounding my porch
 Squid Frigate
 ----
 Potential name for the snack boat business Kate starts inspired by the one on the river in Kyoto
+
+---
```

Comment:
Please follow the established format -- names are offset with sets of four hyphens <3